packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -85,6 +85,10 @@ const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportP
       }
```

Comment:
Could you clean a bit the types in this type? There're some redundant interfaces. 

I was thinking in this direction:

```diff
diff --git a/packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx b/packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx
index 4c6d88235..e9e5558f4 100644
--- a/packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx
+++ b/packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx
@@ -15,15 +15,10 @@ import {
 import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
 import { gridClasses } from '../../gridClasses';
 
-interface GridExportFormat {
-  format: ExportTypes;
-  formatOptions?: GridCsvExportOptions;
-}
-
-type GridExportFormatOption = GridExportFormat;
-
-type GridExportOption = GridExportFormatOption & {
+type GridExportOption = {
   label: React.ReactNode;
+  format: ExportTypes;
+  formatOptions?: GridCsvExportOptions | GridPrintExportOptions;
 };
 
 export interface GridToolbarExportProps extends ButtonProps {
@@ -64,10 +59,10 @@ const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportP
     const handleExport = (option: GridExportOption) => () => {
       switch (option.format) {
         case 'csv':
-          apiRef.current.exportDataAsCsv(option.formatOptions);
+          apiRef.current.exportDataAsCsv(option.formatOptions as GridCsvExportOptions);
           break;
         case 'print':
-          apiRef.current.exportDataAsPrint(option.formatOptions);
+          apiRef.current.exportDataAsPrint(option.formatOptions as GridPrintExportOptions);
           break;
         default:
           break;
```